tests/api2/test_system_dataset.py

```diff
@@ -67,7 +67,7 @@ def test_lock_passphrase_encrypted_pool_with_system_dataset(passphrase_encrypted
 def test_system_dataset_mountpoints():
     system_config = call("systemdataset.config")
     for system_dataset_spec in call(
-        "systemdataset.get_system_dataset_spec", system_config["pool"], system_config["uuid_a"]
+        "systemdataset.get_system_dataset_spec", system_config["pool"], system_config["uuid"]
```

Comment:
I think this is here because the changes were cherry-picked ?
We can workaround this by doing `git rebase master` in your branch if you would like to update your version of the branch from master

Reply:
Yeah that was totally my bad, shouldn't happen in the future!

Reply:
So if we run
```
git rebase --onto 3d4112a3ad475a55bbb5926a134fc0b4281ba526 b86ab7685456b234a7ecc1ec85a2e4e8572b4e60
```
And then force push to this branch, that will effectively remove this commit from your branch and we should be clean on this end ;) 
(the other way would be to do `git rebase -i HEAD~9` and then drop the problematic commit)